shard.yml

```diff
@@ -9,3 +9,8 @@ license: MIT
 targets:
   scry:
     main: src/scry.cr
+
+dependencies:
+  lsp:
+    github: crystal-lang-tools/lsp
+    branch: master
```

Comment:
Instead of pointing to `master` (which can change unexpectedly over time), I think we should point to a tag, for predictable dependencies version

Reply:
All right, I'll point to 0.1 tag, assuming that my MR over at https://github.com/crystal-lang-tools/lsp gets set to that

Reply:
yes probably (hope I don't forget :innocent:)